concent_api/core/tests/test_payments_bankster.py

```diff
@@ -268,20 +268,22 @@ def test_that_settle_overdue_acceptances_should_return_none_if_subtask_costs_whe
             )
         ]
 
-        with mock.patch(
-            'core.payments.bankster.service.get_list_of_payments',
-            side_effect=[
-                self._get_list_of_batch_transactions(),
-                self._get_list_of_force_transactions(),
-            ]
-        ) as get_list_of_payments_mock:
-            claim_against_requestor = settle_overdue_acceptances(
-                requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
-                provider_ethereum_address=task_to_compute.provider_ethereum_address,
-                acceptances=subtask_results_accepted_list,
-                requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
-            )
+        with mock.patch('core.payments.bankster.validate_list_of_transaction_timestamp', return_value=None) as validate_list_of_transaction_mock:
```

Comment:
you doesn't have to provide explicit `return_value` in case you don't expect to get anything out of the function. It is even better not to, because when you set it, it looks as if it was important (an here - it is irrelevant)

Reply:
Done